website/docs/backends/types/remote.html.md

```diff
@@ -91,7 +91,13 @@ set or requires a specific version of Terraform for remote operations, we
 recommend that you create your remote workspaces on Terraform Cloud before
 running any remote operations against them.
 
-## Example Configuration
+## Example Configurations and References
```

Comment:
Ah right. So, that convention in the backends docs of saying "example referencing" or "example reference"... that's always been super weird, and we should probably replace it with a less confusing wording. 

What we're really talking about is how to configure a `terraform_remote_state` data source to use that backend, so how about... (plus second suggestion below, at the relevant header) 

```suggestion
## Example Configurations
```